ansible/roles/hydra/tasks/start.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Install python docker module
+  become: true
+  package:
+    name: python-docker
+    state: present
+
+- name: Create network
+  docker_network:
+    name: "{{ hydra_network }}"
+
+# - name: Create db container
```

Comment:
Not a big fan of commented out code. But I guess it makes sense if we need it later